apis/addons/v1alpha1/addons_types.go

```diff
@@ -67,6 +67,24 @@ type AddonInstallOLMCommon struct {
 	// OLM will resove this package name to install the matching bundle.
 	// +kubebuilder:validation:MinLength=1
 	PackageName string `json:"packageName"`
+
+	// Configs to be passed to subscription OLM object
+	// +optional
+	Config *SubscriptionConfig `json:"config,omitempty"`
+}
+
+type SubscriptionConfig struct {
+	// Array of env variables to be passed to the subscription object.
+	EnvironmentVariables []EnvObject `json:"env"`
+}
+
+type EnvObject struct {
```

Comment:
I didn't use `corev1.EnvVar` since it has an additional `EnvFrom` attribute present and I'm not sure if we want to support that field.

Reply:
I just read https://issues.redhat.com/browse/MTSRE-322 and https://issues.redhat.com/browse/MTSRE-303 and agree: Let's not implement `EnvFrom` :+1: